docs/source/community-participant-recruitment.rst

```diff
@@ -24,7 +24,7 @@ Existing avenues for online outreach:
    -  Interested labs may advertise via the CHS page - we send an invite to be an editor, you may need to double check at https://www.facebook.com/pages/?category=invites to find it. There are known problems about the invites being hard to find: https://www.facebook.com/business/help/community/question/?id=1249301125083191
    -  Experimenting with interest targeting and recording results would be a helpful contribution!
 
-Potential approaches:
+Potential approaches
 =====================
 
 -  Ask participants to tell their friends
```

Comment:
Ignore this suggestion if it is inconvenient: For the section "Potential approaches," are bullets the best way to display this information? Could it be divided into subsections?

Reply:
Thanks for the suggestion. I'm going to keep this as is for now because it was taken directly from Mark / the old CHS site.